src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const result = (days * 40);
+  const magicNumOne = 2;
+  const magicNumTwo = 6;
```

Comment:
😄

Reply:
use more meaningful names
for example - daysForSmallDiscount
and pass amount of discount to variable too
it is better for managing them from one place and for understanding what do they mean